### ISSUE TITLE
Multiple errors preventing starter app from working

### ISSUE THREAD

**Issue by brainlid** [touchstonejs/touchstonejs-starter#45]
The starter project is currently broken. After downloading a fresh install and applying fix in #44, the app still fails to run.

There are many errors. The first of which is rejecting the "Content-Security-Policy".

I reverted the setting here back to the previous one (2 days ago) and the Content-Security-Policy errors went away.

```
<meta http-equiv="Content-Security-Policy" content="default-src 'self'; script-src 'self' * 'unsafe-inline'; img-src * data:; connect-src *; style-src 'self' 'unsafe-inline';">
```

The next error is a failure to fetch cordova.js

```
GET http://localhost:8000/cordova.js
```

I could only get this injected when running the starter from the cordova launcher. To do this I created a new cordova app then copied in the files from the starter. Then added a browser platform (the simplest to get started with for testing). 

```
$ cordova create touchstone-test
$ cordova platform add browser
$ cordova run browser
```

At this point the cordova.js file get added.

When running through ```npm start```, I found I could comment out the cordova.js entry in ```/src/index.html``` to get past it that way.

Then it fails for missing a "screen.js" file. 

I'm looking forward to trying out this framework. I'd appreciate any help you can provide in solving these issues.

Other platform notes:

* Using both Chromium and Google Chrome browser
* Arch Linux
* node 5.1.0
* npm 3.5.0
* cordova 5.4.1

**Comment by JedWatson** [comment]
Odd, it has been working fine for me... the CSP change was due to issues with iOS9, sounds like they're incompatible with the settings needed for Chrome?

Specifically, we need to add the `gap:` protocol to the whitelist or the app fails to start.

I took the new version from the cordova whitelist plugin docs: https://github.com/apache/cordova-plugin-whitelist

Aside from that the 404 on `cordova.js` is "intentional" when you're previewing in the browser; the correct file for the platform is copied into the target `www` folder when you run `cordova prepare` so that's a cordova thing and shouldn't affect the app from running correctly in the browser with `npm start`. Regardless, I think it would be a good idea to explain this in the readme :)

Not sure what's up with the missing screen.js file, I'm going to set up my local environment again from scratch because it sounds like I'm getting different results from you.
